### PR TITLE
[Bug] FlyoutBackgroundImage is not displayed in UWP

### PR DESCRIPTION
**Pull request by danielmonettelli**
### Description of Change ###

I detail that in UWP the "FlyoutBackgroundImage" property fails to observe the image that is at the root of the UWP project.

### Issues Resolved ### 

- fixes #

### API Changes ###

 
 None

### Platforms Affected ### 

- UWP

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 

react-native-gifted-chat
 </a>

### PR Checklist ###

- [x] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

File: EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj [hunk 1]
```diff
@@ -11,9 +11,9 @@
     <AssemblyName>Embedding.UWP</AssemblyName>
     <DefaultLanguage>en-US</DefaultLanguage>
     <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
-    <TargetPlatformVersion>10.0.16299.0</TargetPlatformVersion>
+    <TargetPlatformVersion>10.0.18362.0</TargetPlatformVersion>
```
Comment (jsuarezruiz) [inline EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj]: Why do we need to change the `TargetPlatformVersion`?.

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Views/MainPage.xaml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Views/MainPage.xaml]: This MainPage seems to be a test or something similar. It is not used, could you delete it?.

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems [hunk 3]
```diff
@@ -635,6 +639,10 @@
     <Compile Include="$(MSBuildThisFileDirectory)Issue6994.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)Issue7371.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)ViewModel.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Views\MainPage.xaml.cs">
+      <DependentUpon>MainPage.xaml</DependentUpon>
```
Comment (jsuarezruiz) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Xamarin.Forms.Controls.Issues.Shared.projitems]: Remove also the references to the added MainPage here.

File: Xamarin.Forms.sln [hunk 4]
```diff
@@ -221,7 +221,6 @@ Global
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|ARM.ActiveCfg = Debug|Any CPU
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|ARM.Build.0 = Debug|Any CPU
 		{271193C1-6E7C-429C-A36D-3F1BE5267231}.Debug|iPhone.ActiveCfg = Debug|Any CPU
```
Comment (jsuarezruiz) [inline Xamarin.Forms.sln]: Could you undo this changes in the solution?

File: EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj (hunk 1 above)
Comment (monetelli) [reply EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj]: Hi @jsuarezruiz , I'm new to PRs for Xamarin. Forms, as for the TargetPlatformVersion they force me to change it because I no longer use 10.0.16299 .0.

File: Xamarin.Forms.sln (hunk 4 above)
Comment (monetelli) [reply Xamarin.Forms.sln]: I will @jsuarezruiz , thank you very much for the comments.

File: EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj (hunk 1 above)
Comment (monetelli) [reply EmbeddingTestBeds/Embedding.UWP/Embedding.UWP.csproj]: ... and is ideal since the minimum version is <TargetPlatformMinVersion> 10.0.16299.0 </TargetPlatformMinVersion>

**Comment by jsuarezruiz** [comment]
@monetelli Could you review the conflicts in the PR?. Let us know if we can help you with something!.
